app/pb/communities.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+import "pb/api.proto";
+
+message Point {
+  double lat = 1;
+  double lng = 2;
+}
```

Comment:
This is OK. At some point we will need some more message types for geo data (in a separate file, presumably). There are a few around that we should have a look at.

E.g. `message Polygon { repeated Point = 1; }` would be good.

Reply:
Don't we need a field name in there?

Reply:
Yeah we do :P. My bad.